prow/scripts/cluster-integration/kyma-busola-long-lasting.sh

```diff
@@ -58,14 +67,46 @@ function provisionBusola(){
       --for=condition=ready pod \
       --selector=app.kubernetes.io/component=controller \
       --timeout=120s
+    
+    log::info "Ingress is ready"
+}
+
+function provisionBusola() {
+    export DOMAIN_NAME=$1
+
+    busola_namespace="busola"
+
+    RESOURCES_PATH=${TEST_INFRA_SOURCES_DIR}/prow/scripts/resources/busola/
+
+    log::info "Installing Busola on the cluster: ${DOMAIN_NAME}"
+
+    export KUBECONFIG="${GARDENER_KYMA_PROW_KUBECONFIG}"
+    kubectl get secrets "${DOMAIN_NAME}.kubeconfig" -o jsonpath="{.data.kubeconfig}" | base64 -d > "${RESOURCES_PATH}/kubeconfig--busola--${DOMAIN_NAME}.yaml"
+    export KUBECONFIG="${RESOURCES_PATH}/kubeconfig--busola--$DOMAIN_NAME.yaml"
+
+    # wait for ingress controller to start
+    kubectl wait --namespace kube-system \
```

Comment:
Do we need this here? Why do we need to wait for ingress?

Reply:
Check removed, it was copied to `provisionNginx`